src/app/(app)/api/stripe/webhook/route.ts

```
@@ -0,0 +1,105 @@
+
+          if (
+            !expandedSession.line_items?.data ||
+            !expandedSession.line_items.data.length
+          ) {
+            throw new Error("No line items found");
+          }
+
+          const lineItems = expandedSession.line_items
+            .data as ExpandedLineItem[];
+
+          for (const item of lineItems) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)